src/main/java/pl/edu/mimuw/logic/formulas/single/fFormula.java

```diff
@@ -0,0 +1,27 @@
+package pl.edu.mimuw.logic.formulas.single;
+
+import pl.edu.mimuw.logic.Formula;
+import pl.edu.mimuw.logic.Valuation;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class fFormula implements Formula {
+
+  public fFormula() {}
```

Comment:
niepotrzebne, jak to jedyny konstruktor to jest generowany jako domyślny (musi być bezargumentowy)